debezium-storage/debezium-storage-jdbc/src/main/java/io/debezium/storage/jdbc/JdbcCommonConfig.java

```diff
@@ -27,28 +27,34 @@ public class JdbcCommonConfig {
     private static final Logger LOGGER = LoggerFactory.getLogger(JdbcCommonConfig.class);
 
     public static final String CONFIGURATION_FIELD_PREFIX_STRING = "jdbc.";
+    public static final String CONFIGURATION_FIELD_CONNECTION_GROUP = "connection.";
```

Comment:
```suggestion
    public static final String CONFIGURATION_FIELD_CONNECTION_GROUP = CONFIGURATION_FIELD_PREFIX_STRING + "connection.";
```
I wonder it this would just be a bit more legible because then we can minimize the concatenation in the Field defs below and just use `CONFIGURATION_FIELD_CONNECTION_GROUP`.  wdyt?